test/helpers/client.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
```

Comment:
The next few files are pretty much entirely about spinning up an account and domain for the duration of an integ test.